course_access_groups/models.py

```diff
@@ -4,3 +4,154 @@
 """
 
 from __future__ import absolute_import, unicode_literals
+
+from django.contrib.auth.models import User
+from django.core import exceptions, validators
+from django.db import models
+from django.utils.text import slugify
+from django.utils.translation import ugettext as _
+from model_utils import models as utils_models
+from organizations.models import Organization
+from six import text_type
+
+
+def validate_domain(value):
+    """
+    Validate correct domain name.
+
+    :param value: The domain name.
+
+    This function raise ValidationError when the domain is not valid.
+    """
+    domain_regex = validators.validate_email.domain_regex
+    if not domain_regex.match(value):
+        raise exceptions.ValidationError(_('The domain name is not valid: {domain}').format(domain=value))
+
+
+class CourseAccessGroup(models.Model):
```

Comment:
Why not inherit from `TimeStampedModel`?

Reply:
Done